plugins/backend/catalog-backend-module-okta/package.json

```diff
@@ -6,10 +6,10 @@
   "types": "src/index.ts",
   "license": "Apache-2.0",
   "private": false,
-  "publishConfig": {
-    "access": "public",
-    "main": "dist/index.cjs.js",
-    "types": "dist/index.d.ts"
+  "exports": {
+    ".": "./src/index.ts",
+    "./alpha": "./src/alpha.ts",
```

Comment:
Could you rename this to something else (the underlying file as well)? There's no reason to call this alpha and it will just introduce extra maintenance when it graduates to "beta" or "stable" versions. The naming brings no value this way.

Reply:
I wanted to mimic the situation, which exist in the upstream Backstage plugins. 
There they have the alpha export if the old system is the main one or do you suggest to switch the main export to the new backend system. 

That way the old system would be imported using `@roadiehq/catalog-backend-module-okta/deprecated` and the new backend system using `@roadiehq/catalog-backend-module-okta/deprecated`.